### PR TITLE
change allRegistrationAuthorities name to underscore case

### LINKED ISSUE #509: Rename allRegistrationAuthority URLs
These urls: `allRegistrationAuthorities` need to be converted to underscore case.

### PR DESCRIPTION
**Pull request by drgarcia1986**
FIX #509

and some pep8 issues

### CODE REVIEW

**Comment by LegoStormtroopr** [comment]
Thanks a bunch for this, but do you mind rolling back some of the changes?
We don't check for E501, at least not in `if` statements or method `def`initions.

File: aristotle_mdr/views/views.py [hunk 1]
```diff
@@ -102,18 +105,30 @@ def measure(request, iid, model_slug, name_slug):
 
 
 @cache_per_item_user(ttl=300, cache_post=False)
-def render_if_condition_met(request, condition, objtype, iid, model_slug=None, name_slug=None, subpage=None):
+def render_if_condition_met(
+    request, condition, objtype, iid, model_slug=None,
+    name_slug=None, subpage=None
+):
     item = get_object_or_404(objtype, pk=iid).item
-    if item._meta.model_name != model_slug or not slugify(item.name).startswith(str(name_slug)):
```
Comment (LegoStormtroopr) [inline aristotle_mdr/views/views.py]: Rollback please :)

File: aristotle_mdr/views/views.py [hunk 2]
```diff
@@ -102,18 +105,30 @@ def measure(request, iid, model_slug, name_slug):
 
 
 @cache_per_item_user(ttl=300, cache_post=False)
-def render_if_condition_met(request, condition, objtype, iid, model_slug=None, name_slug=None, subpage=None):
```
Comment (LegoStormtroopr) [inline aristotle_mdr/views/views.py]: Rollback please :)

**Comment by drgarcia1986** [comment]
@LegoStormtroopr For Sure :) Give me a minute

**Comment by drgarcia1986** [comment]
done!
Can you review again @LegoStormtroopr ?

**Comment by LegoStormtroopr** [comment]
Looks good to me. Thanks a bunch :)

**Comment by drgarcia1986** [comment]
you're welcome :smile:

**Pull request merged**